### PR TITLE
Askalono: Upgrade to version 0.4.3

### PR DESCRIPTION
**Pull request by sschuberth**
See https://github.com/jpeddicord/askalono/releases/tag/0.4.3.

Note that Askalono is now distributed in ZIP archives instead of plain
executables.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

File: scanner/src/main/kotlin/scanners/Askalono.kt [hunk 1]
```diff
@@ -91,17 +88,17 @@ class Askalono(name: String, config: ScannerConfiguration) : LocalScanner(name,
...
 
-            val scannerFile = scannerDir.resolve(scannerExe)
-            scannerFile.sink().buffer().use { it.writeAll(body.source()) }
+            log.info { "Unpacking '$archive' to '$unpackDir'... " }
+            body.byteStream().unpack(archive, unpackDir)
 
             if (!Os.isWindows) {
-                // Ensure the executable Unix mode bit to be set.
-                scannerFile.setExecutable(true)
+                // The Linux version is distributed as a ZIP, but without having the Unix executable mode bits stored.
```
Comment (fviernau) [inline scanner/src/main/kotlin/scanners/Askalono.kt:97]: In line 66-70 we distinguish four cases for the OS while here `!Os.isWindows == true` seems to imply `isLinux == true`.
How about mentioning macOs in addition to Linux?

**Review by fviernau — CHANGES_REQUESTED** [review]

File: scanner/src/main/kotlin/scanners/Askalono.kt (hunk 1 above)
Comment (sschuberth) [reply scanner/src/main/kotlin/scanners/Askalono.kt]: That was actually a left-over from copying the code from

https://github.com/oss-review-toolkit/ort/blob/462fcc736fbe62e989b97fb318bac7ee7011f221/scanner/src/main/kotlin/scanners/BoyterLc.kt#L87-L109

and can be removed here, thanks for noticing.

**Review by fviernau — APPROVED** [review]

File: scanner/src/main/kotlin/scanners/Askalono.kt (hunk 1 above)
Comment (fviernau) [reply scanner/src/main/kotlin/scanners/Askalono.kt]: Thanks!

**Comment by sschuberth** [comment]
Merging without waiting for Travis which seems to got stuck.

**Pull request merged**